build-tools/create-dotnet-msi/Microsoft.NET.Workload.Android.wix.in

```diff
@@ -19,6 +19,17 @@
       <ComponentGroupRef Id="ProductComponents" />
     </Feature>
     <WixVariable Id="WixUILicenseRtf" Value="LICENSE.rtf" />
+    <CustomAction Id="dotnet_new_install"   ExeCommand="dotnet.exe new --install &quot;[#ANDROIDTEMPLATES]&quot;" Directory="dotnet" />
+    <CustomAction Id="dotnet_new_uninstall" ExeCommand="dotnet.exe new --uninstall Microsoft.Android.Templates" Directory="dotnet" />
+    <!-- For conditions see: https://stackoverflow.com/a/17608049 -->
+    <InstallExecuteSequence>
+      <Custom Action="dotnet_new_install" After="InstallFinalize">
+        NOT Installed
+      </Custom>
+      <Custom Action="dotnet_new_uninstall" After="InstallFinalize">
+        REMOVE="ALL" AND NOT UPGRADINGPRODUCTCODE
+      </Custom>
+    </InstallExecuteSequence>
```

Comment:
Note that the weird casing here is required 👀 

https://stackoverflow.com/a/17608049